monitoring/roles/loki/files/values.yaml

```diff
@@ -38,7 +38,7 @@ loki:
     enabled: true
     storageClassName: do-block-storage
     accessModes: ["ReadWriteOnce"]
-    size: 50Gi
+    size: 100Gi
```

Comment:
I made this live for prod, but didn't backport to lower envs. I will keep an eye on the lower ones and see if I need to backport it. Also wondering if we'll need more memory for loki/ run it distributed

Reply:
Does distributed mean HA? I think it would be good to be HA, especially for monitoring.

Reply:
Yep, though it is more for the query side than anything

Reply:
The log scraping is a daemonset

Reply:
I can make things HA in a follow up if there's a need